website/docs/prepare-for-merge.md

```diff
@@ -0,0 +1,259 @@
+---
+id: prepare-for-merge
+title: Prepare for The Merge
+sidebar_label: Prepare for The Merge
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[The Merge](https://ethereum.org/en/upgrades/merge/) will fully transition Ethereum's consensus mechanism from proof-of-work to proof-of-stake. This is made possible by the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/), a new Ethereum network layer that implements proof-of-stake consensus. After The Merge, this consensus layer will be fully "merged" with Ethereum's execution layer, and miners will be fully replaced by validators on Ethereum Mainnet.
+
+This guide will walk you through the changes that you need to make in preparation for The Merge.
+
+:::info New user?
+
+This guidance is targeted at users who are already running Prysm. If you're starting from scratch, our [Quickstart](./install/install-with-script.md) will help you configure a Merge-ready node from start to finish.
+
+:::
+
+## The Merge: Before and after
+
+| Before The Merge                                                                                     | After The Merge                                                                                       |
+|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| You don't need to run a local execution client. You can use a service like Infura instead.           | You **do** need to run an execution client. You **can't** use a service like Infura.                  |
+| The HTTP connection between beacon node and execution node doesn't need to be authenticated via JWT. | The HTTP connection between beacon node and execution node **does** need to be authenticated via JWT. |
+| Execution clients don't need to specify a TTD value.                                                 | Execution clients **do** need to specify a TTD value.                                                 |
```

Comment:
Should we omit JWT and also explain what TTD is? I also think this will be automatic and not something users need to do. It is manual for testnets because we need flexibility, but mainnet TTD will be hardcoded into the software as a default

Reply:
Removing both from this table, providing details about JWT below, isolating TTD to just the param update on testnets since this doesn't require any additional user interaction beyond setting the flag